bcipy/helpers/copy_phrase_wrapper.py

```diff
@@ -57,15 +60,26 @@ def __init__(self, min_num_seq, max_num_seq, signal_model=None, fs=300, k=2,
         seq_constants = []
         if backspace_always_shown and BACKSPACE_CHAR in alp:
             seq_constants.append(BACKSPACE_CHAR)
+
+        # Stimuli Selection Module
+        stopping_criteria = CriteriaEvaluator(
+            continue_criteria=[MinIterationsCriteria(min_num_seq)],
+            commit_criteria=[MaxIterationsCriteria(max_num_seq),
+                             ProbThresholdCriteria(decision_threshold)])
+
+        # TODO: Change this hard number 8!
```

Comment:
Are you planning on making this change for the PR?

Reply:
This is another parameter issue. Before stimuli agent, number of stimuli was passed to decision maker. Right now this information should pass to a stimuli agent and then the stimuli agent should be inserted into the decision maker accordingly. I need your input and opinion on this.

Reply:
I just looked back at our previous code and it looks like the stim_length parameter was never used in the Copy Phrase task. We always assume a length of 10. We should create a followup ticket to use that parameter here, but in the meantime maybe just change the value for len_query to 10 for consistency.

Reply:
len_query is hard coded as 10. TODO: Parametrize len_query in the future releases.